libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Arbitrary.hs

```diff
@@ -823,6 +824,12 @@ instance Arbitrary Prices where
 instance Arbitrary CostModel where
   arbitrary = elements nonNativeLanguages >>= genValidCostModel
 
+newtype UnknownCostModels = UnknownCostModels
```

Comment:
Sorry, just a trivial remark: would you mind moving it a bit down, so that it's next to `FlexibleCostModels` ?  I think that would make it easier to find them and understand the difference. 
Also it seems sort of 'changelog-worthy'.

Reply:
We actually don't need a specific type of `UnknownCostModels`. It is much better to use `FlexibleCostModels` , because it also can generate only unknown CostModels, but it can also generate other valid cases. So please remove this type and use `FlexibleCostModels` instead for the test case in this PR
```suggestion
```